src/main/kotlin/org/rust/ide/refactoring/RsRenameProcessor.kt

```diff
@@ -44,4 +46,42 @@ class RsRenameProcessor : RenamePsiElementProcessor() {
     }
 
     private fun String.ensureQuote(): String = if (startsWith('\'')) this else "'$this"
+
+    override fun createUsageInfo(element: PsiElement, ref: PsiReference, referenceElement: PsiElement): UsageInfo {
+        if (ref !is RsStructLiteralFieldReferenceImpl) {
+            return super.createUsageInfo(element, ref, referenceElement)
+        } else {
+            // handle renaming struct literal shorthand identifier
+            val renameSourceType = when (element) {
+                is RsPatBinding -> RenameSourceType.PatBinding
+                is RsNamedFieldDecl -> RenameSourceType.Field
+                else -> null
+            }
+
+            return RenameSourceTypeTrackingUsageInfo(
+                referenceElement,
+                ref,
+                ref.rangeInElement.startOffset,
+                ref.rangeInElement.endOffset,
+                element,
+                ref.resolve() == null && ref.multiResolve(true).isEmpty(),
```

Comment:
Looks like it's enough to call only `ref.multiResolve(true).isEmpty()` here because if `ref.multiResolve(true).isEmpty()` then `ref.resolve()` is always null
Also, I'd prefer to extract this calculation into separate local variable

Reply:
ok